components/indicatorsTable/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from "@/components/uikit/button"
+import { calculateBMI, deleteRecordFromCard } from "@/domains/cards/services"
+import { Card } from "@/domains/cards/types"
+type Props = {
+  id: String
+  card: Card
+}
+
+export const IndicatorsTable = ({ id, card }: Props) => {
+  return (
+    <div>
+      <table className="border border-slate-400 min-w-full mt-5">
```

Comment:
You can create your own `Table` component and add styles in one place. You will get smth like this:

`Table.tsx`
```tsx
import styles from './styles.module.css'

type Props = {
  children: React.ReactNode
  className?: string
}

export const Table = (props: Props) => {
  const { children, className } = props

  return (
      <table
        className={cn(
          'w-full',
          styles.table,
          className,
        )}
      >
        {children}
      </table>
  )
}
```

`styles.module.css`
```styles.module.css
.table th, .table td {
  @apply border border-slate-400;
}
```